doc/source/examples/sharp-immersed-boundary/simple-plane-model-from-composite/simple-plane-model-from-composite.rst

```diff
@@ -0,0 +1,338 @@
+  
+Lets right this file step-by-step Line-by-line:
+
+
+1. First, we create the left wing. This wing is created using a superquadric ellipsoids shape. Along the cord, we use an elongated ellipsoid with lengths 0.6 and 0.1 thickness. In the wingspane direction, the wing is defined by a 6-order power superquadric with a half wingspan of 2.5. We then tilt the wing backward by 0.25 rad and give a small angle of attack of 0.05 rad. Since the wing is now tilted backward, the right side of the superquadric shape is undesirable, so we remove it using a hypercube to trim the right side of the wing.
+
+.. code-block:: text
+
+    shapes
+    0; superquadric; 0.3:2.5:0.05:2:6:2; 0.1:0:0 ; 0:-0.05:0.25 # Full left wing 
+    1; hyper rectangle; 5:5:5;0:-5:0 ; 0:0:0 # Cube to trim of the right side of the left wing
+    operations
+    30; difference  ; 1:0 # Trim of the left wing
+
+This step gives us the first wing of the plane.
+
+.. image:: images/left_wing.png
+   :alt: left_wing
+   :align: center
+   :name: left_wing
+   
+
+2. We repeat the same operation for the right wing and add a union of the and right wing.
+
+.. code-block:: text
+
+    shapes
+    0; superquadric; 0.3:2.5:0.05:2:6:2; 0.1:0:0 ; 0:-0.05:0.25 # Full left wing 
+    1; hyper rectangle; 5:5:5;0:-5:0 ; 0:0:0 # Cube to trim of the right side of the left wing
+    2; superquadric; 0.3:2.5:0.05:2:6:2; 0.1:0:0 ; 0:-0.05:-0.25 # Full right wing 
+    3; hyper rectangle; 5:5:5;0:5:0 ; 0:0:0 # Cube to trim of the left side of the right wing
+    operations
+    30; difference  ; 1:0 # Trim of the left wing
+    31; difference  ; 3:2 # Trim of the right wing
+    32; union       ; 30:31 # Combined the two wings
+
+This step gives us the first wing of the plane.
+
+.. image:: images/both_wing.png
+   :alt: both_wing
+   :align: center
+   :name: both_wing
+
+3. Next, we add the fuselage of the plane. We approximate the fuselage with a circular superquadric shape with a length of 3 and a radius of 0.3. On the length of the plane, the superquadric of power 3. We want the wings to be a bit more at the front of the fuselage, so we move the fuselage slightly backward by 0.25. We then add the combination of the wings and the fuselage.
+.. code-block:: text
+
+    shapes
+    0; superquadric; 0.3:2.5:0.05:2:6:2; 0.1:0:0 ; 0:-0.05:0.25 # Full left wing 
+    1; hyper rectangle; 5:5:5;0:-5:0 ; 0:0:0 # Cube to trim of the right side of the left wing
+    2; superquadric; 0.3:2.5:0.05:2:6:2; 0.1:0:0 ; 0:-0.05:-0.25 # Full right wing 
+    3; hyper rectangle; 5:5:5;0:5:0 ; 0:0:0 # Cube to trim of the left side of the right wing
+    4; superquadric; 1.5:0.3:0.3:3:2:2; -0.25:0:0 ; 0:0:0 # Fuselabe of the plane
+    operations
+    30; difference  ; 1:0 # Trim of the left wing
+    31; difference  ; 3:2 # Trim of the right wing
+    32; union       ; 30:31 # Combined the two wings
+    33; union     ; 32:4 # Combined the wings with the plane fuselage
```

Comment:
```suggestion
    33; union     ; 32:4 # Combine the wings with the plane fuselage
```

Reply:
I think the code block command did not work, maybe it's a question of syntax, but the code looks different than above